worlds/sc2/client.py

```diff
@@ -794,6 +856,11 @@ def locations_for_mission(self, mission: SC2Mission):
         objectives = self.mission_id_to_location_ids[mission_id]
         for objective in objectives:
             yield get_location_offset(mission_id) + mission_id * VICTORY_MODULO + objective
+    
+    def locations_for_mission_id(self, mission_id: int):
```

Comment:
I recommend a typehint for this function. `typing.Iterable[int]` should do, or `typing.Generator[int, None, None]` if you're calling whacky generator-specific functions on the result (which I don't think you are).

Reply:
I copied the existing `locations_for_mission` for this one, which doesn't have a type hint either. I can add it in both places though.